src/command_modules/azure-cli-vm/setup.py

```diff
@@ -32,7 +32,7 @@
 
 DEPENDENCIES = [
     'azure-mgmt-authorization==0.30.0',
-    'azure-mgmt-compute==3.0.1',
+    'azure-mgmt-compute==3.1.0rc1',
```

Comment:
@lmazuel  could you please update the history rst files for `azure-cli-vm` and `azure-cli-acs` as well? That should fix the CI failure